### DISCUSSION TITLE
🔗 New Connections Voting Board

### DISCUSSION THREAD

**Discussion by Gervwyk** [lowdefy/lowdefy#309] [Ideas and feature requests]
Hi Everyone.

This topic is created to get input from the Lowdefy community over which Lowdefy connections should be added to the Lowdefy. 

Please post which connection you'd like to see supported, and get others to upvote your comment. First check if the connection you are looking for is already requested and upvote to consolidate votes, else add a new comment with one connection per comment.

We will give priority to the highest voted requests, although we will also take into account technical consideration when deciding what connection to develop next. 

Our plan is to enable custom Lowdefy connections in the near future.

**Comment by Gervwyk** [comment]
#### Released in [v3.13.0](https://github.com/lowdefy/lowdefy/releases/tag/v3.13.0).
MySQL

See: https://docs.lowdefy.com/MySQL

**Reply by SamTolmay** [reply]
Released in [v3.13.0](https://github.com/lowdefy/lowdefy/releases/tag/v3.13.0).

**Comment by Gervwyk** [comment]
Github

**Reply by astrogewgaw** [reply]
This would be awesome 😄 , especially because what I have in mind for my first **lowdefy** app requires putting data in, and getting data from, a GitHub repository 👍🏾 .

**Reply by SamTolmay** [reply]
I have created a discussion (#474) where we can discuss implementation ideas for a Github connection. Please share what you would want to do with a Github connection, and how you would like the connection to work [here](https://github.com/lowdefy/lowdefy/discussions/474).

**Comment by Gervwyk** [comment]
#### Released in [v3.13.0](https://github.com/lowdefy/lowdefy/releases/tag/v3.13.0).
PostgreSQL

See: https://docs.lowdefy.com/PostgreSQL

**Reply by SamTolmay** [reply]
Released in [v3.13.0](https://github.com/lowdefy/lowdefy/releases/tag/v3.13.0).

**Reply by alexander-mart** [reply]
@SamTolmay please update on site main page — remove "Comming Soon" under PostgreSQL and other: https://lowdefy.com

**Comment by Gervwyk** [comment]
Shopify

**Comment by InvncibiltyCloak** [comment]
SMTP

**Comment by thriqon** [comment]
Kubernetes Custom Resources

**Comment by rblalock** [comment]
Firebase

**Reply by gustavopch** [reply]
I suggest editing the comment to "Firebase" (which is the correct way to write it) instead of "Fire base" because currently, this comment doesn't appear in the search results for "Firebase".

**Comment by xuoguoto** [comment]
Websockets

**Comment by astrogewgaw** [comment]
#### Released in [v3.13.0](https://github.com/lowdefy/lowdefy/releases/tag/v3.13.0).
SQLite

See: https://docs.lowdefy.com/SQLite

**Reply by SamTolmay** [reply]
Released in [v3.13.0](https://github.com/lowdefy/lowdefy/releases/tag/v3.13.0).

**Comment by phillc73** [comment]
DuckDB

**Comment by SamTolmay** [comment]
Xero Accounting

**Comment by cveenstra** [comment]
BigQuery

**Comment by AhmedBytesBits** [comment]
Elasticsearch

**Reply by Gervwyk** [reply]
Released in [v.3.19.0](https://github.com/lowdefy/lowdefy/releases/tag/v3.19.0). See https://docs.lowdefy.com/Elasticsearch

**Comment by devtombiz** [comment]
Neo4j
RedisGraph

**Comment by vaddisrinivas** [comment]
- Redis
- local Excel file support 
- WYSIWYG editor for markdown

**Comment by m8e** [comment]
LuckySheet (self-hosted Excel365)

**Comment by codexetreme** [comment]
GRPC
can be used for a lot internal tools that exclusively provide gRPC apis

**Comment by meier-christoph** [comment]
Trino
https://trino.io/

**Comment by 13316090972** [comment]
mqtt

**Comment by zainhoda** [comment]
GraphQL

**Comment by covalesj** [comment]
Generic jdbc connection.

**Reply by ClorisYT** [reply]
It is very crucial!!

**Comment by kfoxmchd** [comment]
Supabase.

**Comment by Dock1100** [comment]
Clickhouse

**Comment by ehangreens** [comment]
Sharepoint?

**Reply by SamTolmay** [reply]
Hi @ehangreens 

What use case are you looking at with a sharepoint connection? What type of functionality would you need?

**Reply by ehangreens** [reply]
Hi @SamTolmay ,

Thanks for your response, really only an excel file hosted on sharepoint

**Reply by ArmandDes** [reply]
I think it would be great to connect to a SharePoint online list also

**Reply by iron3oxide** [reply]
> Hi @ehangreens
> 
> What use case are you looking at with a sharepoint connection? What type of functionality would you need?

To chime in here: I have a use case where I want to create a list item (I think that's the SP term for what is basically a table row) on a sharepoint server from within a lowdefy app in order to have the input data available to both of them. Since I don't think one can pass field values via url to SP, one would have to connect properly, authentication and all.

One way I can think of to create this connection in lowdefy may be to wrap (at least parts of) [pnpjs](https://pnp.github.io/pnpjs/) somehow. Does that seem feasible?

**Comment by ehangreens** [comment]
Sage pastel ! will be a complete game changer in my world

**Comment by anthonyangel** [comment]
More AWS services:
- Connection to DynamoDB
- Trigger Lambdas / Step Functions
- EventBridge
- AWS authentication based on IAM role assumption rather than static API keys

**Comment by bregy2** [comment]
A Basic fileInput Component? Not S3.. 
Is there a reason/limitation why it isn't already implemented?

**Comment by Dave-Wagner** [comment]
[Appwrite](https://appwrite.io/) would be nice, especially with real-time support.

**Comment by kemeris2000** [comment]
WebSocket
